data/src/main/java/co/nimblehq/compose/crypto/data/model/response/CoinItemResponse.kt

```diff
@@ -72,7 +72,7 @@ data class CoinItemResponse(
     )
 }
 
-private fun CoinItemResponse.toModel() = CoinItem(
+fun CoinItemResponse.toModel() = CoinItem(
```

Comment:
How about keep this extension `internal`? 🤔

Reply:
@hoangnguyen92dn Yes, however I think you can review this in our template instead https://github.com/nimblehq/android-templates/pull/311/files#diff-2ab311aa3921aeffb9dd9eb349a8ec774b2e63220b2793c49db434d68d0c52b2R15 😛

Reply:
@hoangnguyen92dn Anw, fixed in https://github.com/nimblehq/jetpack-compose-crypto/pull/43/commits/3f8d5dacdc2bc50a6e483c09633968c55a2a6034